### ISSUE TITLE
proto message using map NoSuchMethodError

### ISSUE THREAD

**Issue by SeriousMa** [grpc/grpc-java#3149]
it seems invoke  CodedOutputStream.class in grpc-protobuf-lite 
![image](https://user-images.githubusercontent.com/464213/27632080-3a040c34-5c2d-11e7-8780-27aa1b111bc9.png)
![image](https://user-images.githubusercontent.com/464213/27632085-3e686b9e-5c2d-11e7-815a-3a9c32fb199c.png)
![image](https://user-images.githubusercontent.com/464213/27632086-411628f4-5c2d-11e7-8b05-58bde4582b3a.png)

```
  <dependency>
                <groupId>io.grpc</groupId>
                <artifactId>grpc-all</artifactId>
                <version>1.3.0</version>
            </dependency>
```

**Comment by ejona86** [comment]
Maybe you're hitting grpc/grpc-java#2985, since `isSerializationDeterministic` is not in protobuf-lite. Could you define your deps without grpc-all and grpc-protobuf-lite and see if that resolves the problem?

**Comment by SeriousMa** [comment]
i exclusion  protobuf-lite and it's ok now
```
 <dependency>
                <groupId>io.grpc</groupId>
                <artifactId>grpc-all</artifactId>
                <version>${grpc.version}</version>
                <exclusions>
                    <exclusion>
                        <artifactId>protobuf-lite</artifactId>
                        <groupId>com.google.protobuf</groupId>
                    </exclusion>
                </exclusions>
            </dependency>
```

**Comment by ejona86** [comment]
Fixed by #3153
